### PR TITLE
Fix inject into file warning

### LINKED ISSUE #706: insert_into_file print warning when change is already done
If I use `append_file` or `insert_into_file` in a generator, and I execute generator twice, then `File unchanged! The supplied flag value not found!` warning is printed.

Is this the expected behavior. It sounds curious to me. Why doesn't it print `identical` or `unchanged` without warning, since `content.include?(replacement)` returns true?

Related to #683

### PR DESCRIPTION
**Pull request by nicolas-brousse**
Not sure if the fix implementation is good enough. Let me know if you want some changes.

Closes #706.

### CODE REVIEW

**Review by dorner — APPROVED** [review]
This looks good - I'd like to see this merged in!

**Comment by dorner** [comment]
@nicolas-brousse can you rebase off master please?

File: lib/thor/actions/inject_into_file.rb [hunk 1]
```diff
@@ -96,22 +98,32 @@ def say_status(behavior, warning: nil, color: nil)
...
         super(status, (color || config[:verbose]))
       end
 
+      def content
+        @content ||= File.read(destination)
+      end
+
+      def replacement_present?
+        before, after = content.split(regexp, 2)
+        snippet = (behavior == :after ? after : before).to_s
```
Comment (nicolas-brousse) [inline lib/thor/actions/inject_into_file.rb:116]: This code has been added. I've to found a way to deal this `regexp`.

**Review by dorner — APPROVED** [review]
Looks good! @rafaelfranca

**Comment by nicolas-brousse** [comment]
For now when I tried locally, the test I added fail. I've to inspect how `snippet = (behavior == :after ? after : before).to_s` works. When I initially worked on this PR this line wasn't implemented yet.

**Pull request merged**